docker-compose.yml

```diff
@@ -1,7 +1,38 @@
 version: '3'
 services:
-  dev:
+  app: &main
+    image: ex-data-telemetry
     build: .
+    volumes:
+      - ./data:/data
+    environment:
+      MYSQL_DB_HOST: mysql
+      MYSQL_DB_USER: root
+      MYSQL_DB_PASSWORD: rootpassword
+      MYSQL_DB_DATABASE: test
+      MYSQL_DB_PORT: 3306
+      KBC_PROJECTID: 2
+      KBC_STACKID: connection.north-europe.azure.keboola.com
+    links:
+      - mysql
+  dev:
+    <<: *main
     volumes:
       - ./:/code
       - ./data:/data
+
+  mysql:
+    image: mysql:5.6
+    environment:
+      MYSQL_DATABASE: test
+      MYSQL_ROOT_PASSWORD: rootpassword
+    ports:
+      - 3306:3306
+
+  wait:
+    image: waisbrot/wait
+    depends_on:
+      - mysql
+    environment:
+      - TARGETS=mysql:3306
+      - TIMEOUT=200
```

Comment:
Tip: nastav si IDCko aby nikdy nechybal novy riadok na konci suboru
![image](https://user-images.githubusercontent.com/19371734/98829052-0cb73080-2439-11eb-9b79-e1d1b65cfde6.png)

Reply:
jo to už bych asi mohl :)